net/acme/files/run.sh

```diff
@@ -335,7 +335,17 @@ issue_cert()
 		# commit and reload is in post_checks
 	fi
 
-	if [ -e /etc/init.d/nginx ] && [ "$update_nginx" -eq "1" ]; then
+	nginx_updated=0
```

Comment:
This should be declared `local` somewhere :)

Reply:
done :-)

Reply:
Declaring a variable local and assigning to it on the same line is a bashism, unfortunately... doesn't work on the busybox shell :(

Reply:
Oh, shellcheck.net was not complaining (for #!/bin/ash) ... Changed it now, too. Thanks.